README.md

```diff
@@ -26,10 +22,14 @@ angular.module('App', [
 
 ## Related
 
-* [Web Control Panel UI](https://github.com/ovh-ux/ovh-manager-web).
-* [Emailpro Module](https://github.com/ovh-ux/ovh-module-emailpro).
-* [Exchange Module](https://github.com/ovh-ux/ovh-module-exchange).
-* [Office Module](https://github.com/ovh-ux/ovh-module-office).
+* [ovh-manager-web](https://github.com/ovh-ux/ovh-manager-web) - OVH Control Panel Web UI
+* [ovh-module-emailpro](https://github.com/ovh-ux/ovh-module-emailpro) - Web Module Emailpro
+* [ovh-module-exchange](https://github.com/ovh-ux/ovh-module-exchange) - Web Module Exchange
+* [ovh-module-office](https://github.com/ovh-ux/ovh-module-office) - Web Module Office
+
+## Contributing
+
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/ovh-module-sharepoint/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/ovh-module-sharepoint/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

Comment:
... [filling ... (typo)

Reply:
I'm not sure about it (https://www.thefreedictionary.com/filing)